src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod config;
```

Comment:
Ditto
```suggestion
```

Reply:
can't access 'src/config.rs ' under the `tests` directory.
How about the first of the following methods?
1. change accessibility to `pub` for accessible in tests directory
`mod config.rs` -> `pub mod config.rs`
4. move the test file in src directory
src/config.rs
tests/config.rs -> src/config_test.rs